metricbeat/module/prometheus/_meta/fields.yml

```diff
@@ -1,9 +1,9 @@
 - key: prometheus
   title: "Prometheus"
   description: >
-    Prometheus Module
-
-    experimental[]
+    beta[]
```

Comment:
We should be careful not to change "history" as in the code it is still "experimental". I think it is fine here as we kind of "meant" beta back then but it didn't exist yet.